packages/docusaurus-plugin-openapi-docs/src/openapi/utils/loadAndResolveSpec.ts

```diff
@@ -26,6 +27,18 @@ function serializer(replacer: any, cycleReplacer: any) {
     };
 
   return function (key: any, value: any) {
+    // Resolve discriminator ref pointers
+    if (value.discriminator !== undefined) {
```

Comment:
```
Loading of api failed for "/Users/blinda/Desktop/pan-docusaurus/docusaurus-openapi/demo/examples/code/CodeSecurity.yaml"
[ERROR] TypeError: Cannot read properties of null (reading 'discriminator')
```
So far tested with the following specs: 
- CSPM 
- CWPP 
- Code Security 
- IoT
- ATP/TP

As for the error message when generating some of the api docs, I added an optional chain in the case that `value` evaluates to null.

`if (value?.discriminator !== undefined) {`
 
 @sserrata I can push the fix up to the same branch if that's fine 👍

Reply:
Absolutely, thanks for catching this!